migrations/src/main/scala/za/co/absa/enceladus/migrations/continuous/migrate01/DatasetMigrator.scala

```diff
@@ -52,7 +52,7 @@ final class DatasetMigrator(evm: EntityVersionMap,
     */
   def migrateEntity(srcDatasetJson: String, objectId: String, repo: EntityRepository): Unit = {
     Try {
-      val fixJson = srcDatasetJson.replaceAll("\"jsonClass\" :", "\"_t\" :")
+      val fixJson = srcDatasetJson.replaceAll("\"jsonClass\"\\w*:", "\"_t\" :")
```

Comment:
small `w` is `[a-zA-Z0-9_]` which should never occur between the `"jsonClass"` key and `:`

Reply:
lol, it's what I get for being hasty.
Thanks!